core/src/test/resources/logback-test.xml

```diff
@@ -25,4 +25,5 @@
         <appender-ref ref="STDOUT"/>
     </root>
     <logger name="com.datastax.oss.driver" level= "${driverLevel:-WARN}"/>
+    <logger name="Tmp" level= "DEBUG"/>
```

Comment:
What is this logger for?

Reply:
Hehe, that's an accidental commit, fixing.